app/uk/gov/hmrc/identityverificationquestions/sources/empRef/EmpRefService.scala

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.identityverificationquestions.sources.empRef
+
+import uk.gov.hmrc.identityverificationquestions.config.AppConfig
+import uk.gov.hmrc.identityverificationquestions.connectors.QuestionConnector
+import uk.gov.hmrc.identityverificationquestions.models._
+import uk.gov.hmrc.identityverificationquestions.monitoring.EventDispatcher
+import uk.gov.hmrc.identityverificationquestions.monitoring.auditing.AuditService
+import uk.gov.hmrc.identityverificationquestions.services.utilities.{CheckAvailability, CircuitBreakerConfiguration, PenceAnswerConvertor, TaxYearBuilder}
+import uk.gov.hmrc.identityverificationquestions.sources.QuestionServiceMeoMinimumNumberOfQuestions
+
+import javax.inject.{Inject, Singleton}
+
+@Singleton
+class EmpRefService @Inject()(empRefConnector: EmpRefConnector, val eventDispatcher: EventDispatcher, val auditService: AuditService)(implicit override val appConfig: AppConfig)
+  extends QuestionServiceMeoMinimumNumberOfQuestions
+    with CheckAvailability with CircuitBreakerConfiguration with TaxYearBuilder with PenceAnswerConvertor {
```

Comment:
We should have a it:test for the circuit breaker logic

Reply:
the circuit breaker test in super class